_rules/SC2-4-6-descriptive-headings.md

```diff
@@ -94,7 +101,7 @@ Heading marked up with `h` element that is a single character that describes the
 
 #### Passed example 5
 
-Heading marked up with `role="heading"` that describes the topic or purpose of its section of the content. The heading is positioned off screen and is included in the Accessibility Tree.
+The heading marked up with `role="heading"` describes the topic or purpose of its section of the content. The heading is positioned off screen and is included in the Accessibility Tree.
```

Comment:
What if the heading is visible and included in the accessibility tree and the section is positionned off screen ?

````
<span role="heading">Opening Hours</span>
<p style="position: absolute; top: -9999px; left: -9999px;">We are open Monday through Friday from 10 to 16</p>
````

Would you consider it a passed example ? Or does it fail into the case of "no section of the content" since the content is not visible ?

Is the expectation 2 valid in that configuration ? 
 
> The visible content, if there is any, of each target element describes the topic or purpose of its section of the content in part or in its entirety, or the heading has no section of the content.

Don't we need to be a little more precise on the visibility/state of the "section of the content", as you've done perfectly for the heading ? 

Does the "section of the content" has to be "visible" ? Or does a "section of the content" only need to be in the accessibility tree ? 
and... what if the section of the content is visible but not included in the accessibility tree ?

Reply:
@audreymaniez, I think you are perfectly right about specifying if the content has to be visible or included in the accessibility tree. 
However, the "no section of content" makes everything pass anyway, so I think we need to have another look at that part too. I haven't updated the test cases yet, since a decision on the "no section content" part can change the test cases quite a lot.